applications/callflow/src/module/cf_set_alert_info.erl

```diff
@@ -0,0 +1,37 @@
+%%%-----------------------------------------------------------------------------
+%%% @copyright (C) 2013-2018, 2600Hz
+%%% @doc "data":{
+%%%   "alert_info":{{value}},
+%%% }
+%%%
+%%%
+%%% @author Sergey Safarov <s.safarov@gmail.com>, Sponsored by Audian
+%%% @end
+%%%-----------------------------------------------------------------------------
+-module(cf_set_alert_info).
+
+-behaviour(gen_cf_action).
+
+-include("callflow.hrl").
+
+-export([handle/2]).
+
+%%------------------------------------------------------------------------------
+%% @doc Entry point for this module
+%% @end
+%%------------------------------------------------------------------------------
+-spec handle(kz_json:object(), kapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    Value = kz_json:get_ne_binary_value(<<"alert_info">>, Data),
+    set_variable(Value, Call),
+    Call1 = kapps_call:insert_custom_channel_var(<<"Alert_info">>, Value, Call),
```

Comment:
Why not just use `kapps_call:set_custom_channel_var/3` which should set the var on the channel and update the call record's CCVs?

Reply:
Hello James
Your suggestions is applied.